scipy/fftpack/tests/test_basic.py

```diff
@@ -199,6 +199,28 @@ def test_notice(self):
         pass
 
 
+class TestFloat16FFT(TestCase):
+    def setUp(self):
+        self.cdt = np.complex64
+        self.rdt = np.float16
```

Comment:
If you don't inherit from `_TestFFTBase`, there's no need for this.
Do you need to inherit from it?